TeamCode/src/main/java/org/firstinspires/ftc/teamcode/helpers/utils/GlobalConfig.java

```diff
@@ -0,0 +1,37 @@
+package org.firstinspires.ftc.teamcode.helpers.utils;
+
+import com.acmerobotics.dashboard.config.Config;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+@Config
+public class GlobalConfig {
+    private static boolean DEBUG_MODE = false;
+
+    private static final Map<ConfigVariable, Object> defaultValueMap = new HashMap<>();
```

Comment:
extract all logic an abstract class, which the `GlobalConfig` one implements maybe? it doesn't really make sense to have the logic in the same class as the actual config